scriptworker/config.py

```diff
@@ -134,11 +114,12 @@ def check_config(config, path):
     """
     messages = []
 
-    missing_keys = set(DEFAULT_CONFIG.keys()) - set(config.keys())
+    config_copy = get_frozen_copy(config)
```

Comment:
Nit: Shouldn't we create the frozen copy in `create_config()`? This would enable to (ask to) delete the mutable copy there.